design/source/components/Asset-Editor/_asset-editor.scss

```diff
@@ -0,0 +1,15 @@
+@import 'general';
+@import 'atoms/asset-input/asset-input';
```

Comment:
I wonder whether we should put those styles into the design or not rather into the editor. I know that one doesn't want to mix editor styles with styles from the design but to be fair, this is the case here anyway: we do an asset editor in the document.
The consequence of doing it in the design is just that we will copy / paste it for every new design. Next example: we also want this for heute. There we would need plain CSS since they don't use SASS. So we would need to compile and then copy / paste it there. Doesn't seem too nice either...

@peyerluk